compiler/quilt/test/test_command.py

```diff
@@ -18,6 +18,106 @@
 from .utils import QuiltTestCase, patch
 
 class CommandTest(QuiltTestCase):
+    # no counterpart to command.get_registry_url, so we mock _save_config and _load_config
+    @patch('quilt.tools.command._save_config')
+    @patch('quilt.tools.command._load_config')
+    @patch('quilt.tools.command.input')
+    def test_config_urls_default(self, mock_input, mock_load_config, mock_save_config):
+        config_prompt = ("Please enter the URL for your custom Quilt registry (ask your administrator),\n" +
```

Comment:
OK by me.

Reply:
Well, I'll have to use 'assert mock_input.called',  as 'mock_input.assert_called()' doesn't seem to work when testing.  It might be that it was added in a later version.